clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/PersistenceDAO.scala

```diff
@@ -168,7 +218,15 @@ object PersistenceDAO {
     * aren't really a thing in GCS. We could special-case the local-file DAO but
     * the extra complexity isn't worth it.
     */
-  val versionFileName = "current-clio-version.txt"
+  val VersionFileName = "current-clio-version.txt"
+
+  /**
+    * Directory depth to naively walk during recovery.
+    *
+    * Should be deep enough to cover date-based partitions of documents in storage,
+    * but not the documents themselves.
+    */
+  val StorageWalkDepth = 3
```

Comment:
We should have a comment somewhere that describes the path elements in our Persistence storage, which will make this value less magic. If we have it already, link to it from here, or add it here if we don't have it already.

Reply:
Done